app/helpers/settings_schedule_helper.rb

```diff
@@ -0,0 +1,118 @@
+module SettingsScheduleHelper
+  private
+
+  def settings_schedule_summary(schedule, exp_table, object_class, object_name)
+    automation_request_check = schedule.sched_action[:method] == 'automation_request'
+    summary = [
+      settings_schedule_basic_info(schedule),
+      settings_schedule_filter_details(schedule, exp_table),
+    ]
+    summary.push(settings_schedule_object_details(schedule)) if automation_request_check
+    summary.push(settings_schedule_object_attribute(object_class, object_name)) if automation_request_check
+    summary.push(settings_schedule_attribute_value_pairs(schedule)) if automation_request_check && schedule.filter[:ui][:ui_attrs].present?
```

Comment:
The `do_something if condition` is great for single items but if you have multiple it is best to group under a standard conditional
```suggestion
    if automation_request_check
      summary.push(settings_schedule_object_details(schedule)) 
      summary.push(settings_schedule_object_attribute(object_class, object_name))
      summary.push(settings_schedule_attribute_value_pairs(schedule)) if schedule.filter[:ui][:ui_attrs].present?
    end
```

Reply:
resolved